README.md

```diff
@@ -125,7 +125,7 @@ Save them to your `.bash_profile` or equivalent.
 1. `python manage.py createsuperuser`
 2. `python manage.py runserver` to run a development server on localhost:8000 or see below for how to deploy via Apache.
 3. Visit http://server.ip/admin and log in with new user.
-4. You should see a list of content. _Don't try to directly edit data that came from the shapefiles - they are liable to be so complex with so many points that attempting this freezes or crashes the browser._ **TODO: Do those shapefiles need to show up in here at all?**
+4. You should see a list of content. _Don't try to directly edit data that came from the shapefiles/rasters - they are liable to be so complex with so many points that attempting this freezes or crashes the browser._ **TODO: Do those shapefiles need to show up in here at all?**
 5. There are other pieces of content that you can and should edit, though! They are bits of text and other information that show up on this site.
```

Comment:
No, they don't. I could take them out of django admin and then we wouldn't have this problem.